src/Imath/half.h

```diff
@@ -193,27 +201,31 @@ class half
 //-------------------------------------------------------------------------
 // Limits
 //
-// Visual C++ will complain if HALF_MIN, HALF_NRM_MIN etc. are not float
+// Visual C++ will complain if HALF_DENORM_MIN, HALF_NRM_MIN etc. are not float
 // constants, but at least one other compiler (gcc 2.96) produces incorrect
 // results if they are.
 //-------------------------------------------------------------------------
 
 #if (defined _WIN32 || defined _WIN64) && defined _MSC_VER
 
-#    define HALF_MIN 5.96046448e-08f // Smallest positive half
+#    define HALF_DENORM_MIN 5.96046448e-08f // Smallest positive denormalized positive half
 
 #    define HALF_NRM_MIN 6.10351562e-05f // Smallest positive normalized half
 
+#    define HALF_MIN 6.10351562e-05f // Smallest positive normalized half
```

Comment:
I agree this is better nomenclature, but would it be better to eliminate HALF_MIN altogether rather than changing its value? The value is available as HALF_NRM_MIN.  Any current use of HALF_MIN deserves careful inspection, so it may be better if it produces an error.

Reply:
I guess this is mimicking C's FLOAT_MIN, FLOAT_MAX.

Reply:
Also, I was wondering if we should eliminate the vector/etc classes having BaseTypeMax() methods, et al. There was some confusing nomenclature, and there's no real reason for this, versus using `std::numeric_limits<VecType::BaseType>::max()` or whatever. The reason why I want to remove is, again, historical differences between Imath nomenclature and std. Maybe better to just remove it and any confusion.